src/admin/shared/helpers/render-detail-fields.tsx

```diff
@@ -35,11 +35,13 @@ export function renderSimpleDetailRows(
 
 export function renderDateDetailRows(
 	obj: any,
-	propAndTranslations: [string, string][]
+	propAndTranslations: ([string, string] | [string, string, string])[]
 ): ReactElement[] {
 	return propAndTranslations.map(propAndTranslation => {
 		return renderDetailRow(
-			obj[propAndTranslation[0]] ? formatDate(obj[propAndTranslation[0]]) : '-',
+			obj[propAndTranslation[0]]
+				? formatDate(obj[propAndTranslation[0]])
+				: propAndTranslation[2] || '-',
```

Comment:
best nog in de jsdoc voor deze functie vermelden wat er in de propAndTranslation[2] zit